frontend/src/pages/org/archived-item-qa.ts

```diff
@@ -118,43 +156,144 @@ export class ArchivedItemQA extends TailwindElement {
+                id="replay-tab"
+                href=${`${crawlBaseUrl}/review/replay`}
+                ?active=${this.tab === "replay"}
+                @click=${this.navigate.link}
+              >
+                ${msg("Replay")}
+              </btrix-navigation-button>
+            </div>
+            <div class="flex gap-4">
+              <sl-button size="small">
+                <sl-icon slot="prefix" name="arrow-left"></sl-icon>
+                ${msg("Previous Page")}
+              </sl-button>
+              [btrix-page-qa-toolbar]
+              <sl-button variant="primary" size="small">
+                <sl-icon slot="suffix" name="arrow-right"></sl-icon>
+                ${msg("Next Page")}
+              </sl-button>
+            </div>
           </nav>
-          <div role="region" aria-labelledby="${this.tab}-tab">
-            ${choose(
-              this.tab,
-              [
-                ["screenshots", this.renderScreenshots],
-                ["replay", this.renderReplay],
-              ],
-              () => html`<btrix-not-found></btrix-not-found>`,
-            )}
-          </div>
+          ${this.renderToolbar()} ${this.renderSections()}
         </section>
         <h2 class="pageListHeader outline">
           ${msg("Pages List")} <sl-button>${msg("Finish Review")}</sl-button>
         </h2>
-        <section class="pageList outline">[page list]</section>
+        <section class="pageList outline">
+          <ul>
+            ${this.pages?.items.map(
+              (page) => html`
+                <li>
+                  <a
+                    class="underline"
+                    href="${window.location.pathname}?itemPageId=${page.id}"
+                    @click=${this.navigate.link}
+                  >
+                    id: ${page.id}</a
+                  >
+                </li>
+              `,
+            )}
+          </ul>
+          pg ${this.pages?.page} of
+          ${this.pages
+            ? Math.ceil(this.pages.total / this.pages.pageSize)
+            : "unknown"}
+        </section>
       </article>
     `;
   }
 
+  private renderToolbar() {
+    return html`
+      <div
+        class="${this.tab === "replay"
+          ? "rounded-t-lg"
+          : "rounded-lg"} my-2 flex h-12 items-center border bg-neutral-50 text-base"
+      >
+        <div class="ml-1 flex">
+          ${choose(this.tab, [
+            [
+              "replay",
+              () => html`
+                <sl-icon-button name="arrow-clockwise"></sl-icon-button>
+              `,
+            ],
+            [
+              "screenshots",
+              () => html`
+                <sl-icon-button name="intersect"></sl-icon-button>
+                <sl-icon-button name="vr"></sl-icon-button>
+              `,
+            ],
+          ])}
+        </div>
+        <div
+          class="mx-1.5 flex h-8 flex-1 items-center justify-between rounded border bg-neutral-0 px-2 text-sm leading-none"
+        >
```

Comment:
```suggestion
          class="mx-1.5 flex h-8 min-w-0 flex-1 items-center justify-between rounded border bg-neutral-0 px-2 text-sm leading-none"
```

Reply:
Looks like this got offset by 1 line for some reason — it's just adding `min-w-0` after `h-8`